_layouts/trademarks.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+---
+<!--
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+-->
+<div id="whiskIndexedLayout">
+
+    <!-- Community Index -->
+    <div id="whiskIndex">
+        <ul>
+            <li>Trademark Guidelines</li>
+
+        </ul>
+    </div>
+
+
+    <section id="whiskNodes">
+        <main class="doc">
+            <div class="content">
+                <a class="indexable" id="trademarks"></a>
+                <h2>Apache OpenWhisk Trademark Guidelines</h2>
+            </div>
+        </main>
+
+        <main class="doc">
+            <div class="content">
+
+                <p>Apache OpenWhisk, OpenWhisk and the OpenWhisk logo are
+                <a href="https://www.apache.org/foundation/marks/">trademarks</a>
+                of the <a href="https://www.apache.org/">Apache Software Foundation (ASF)</a>.
+                The ASF is a 501(c)(3) nonprofit organization, and as such, needs to take
+                special care about how its trademarks are used by organizations.
+                In particular, ASF needs to ensure that its software products are clearly
+                distinguished from third-party products.</p>
+
+                <p>If you would like to provide software, services, events, or other products
+                based on Apache OpenWhisk, please refer to the
+                <a href="https://www.apache.org/foundation/marks/">ASF trademark policy</a>
+                and <a href="https://www.apache.org/foundation/marks/faq/">FAQ</a>.
+                This page summarizes the key points of the policy, but please note that
+                the official policy always takes precedence.</p>
+
+                <ul>
+                  <li><strong>Software products,</strong> whether commercial or open source, are not allowed to use “OpenWhisk” in their name, except in the form “powered by Apache OpenWhisk" or “for Apache OpenWhisk” when following
+                <a href="https://www.apache.org/foundation/marks/faq/#products">these specific
+                guidelines</a>.
+                    <ul>
+                      <li>Names derived from “OpenWhisk” are also not allowed.</li>
+                      <li>Company names may not include “OpenWhisk”.</li>
+                      <li>Package identifiers (e.g., Maven coordinates) may include
```

Comment:
I know this is inspired by the spark example - but when i read this there and now here, i found it confusing. What is the "naming policy above" and what is an allowed package name?

Reply:
@jvz suggested on the dev list:

```
Good idea to define naming schemes like how maven does for plugins. For
example, the official project plugins follow a naming scheme of
maven-foo-plugin, while third party plugins follow a naming scheme of
foo-maven-plugin. Similar to the distinction between “Apache Maven Foo
Plugin” versus “Foo Plugin for Apache Maven”.
```

I like this and maybe can resolve my concern. Happy to PR if I get some agreement.

Reply:
i think it revolves around any use of "openwhisk" in any URIs (paths) or in system path/classpath relative path names referenced in modules/tooling etc. is allowed; that is, people can reference "openwhisk" when building from any libraries with the name "openwhisk" (that we most likely published as part of our source code or resultant binaries from builds steps).   That was what I read... did not confuse me as Maven build files often point to repose/modules with the project names in them (which would be allowed).